src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<? extends Machine> get() {
+        return List.of(new Bulldozer(),new Bulldozer(), new Bulldozer());
```

Comment:
when you added some fields you need to give those as parameters for some objects